modules/imgproc/perf/perf_goodFeaturesToTrack.cpp

```diff
@@ -6,15 +6,16 @@ using namespace perf;
 using std::tr1::make_tuple;
 using std::tr1::get;
 
-typedef std::tr1::tuple<string, int, double, int, bool> Image_MaxCorners_QualityLevel_MinDistance_BlockSize_UseHarris_t;
-typedef perf::TestBaseWithParam<Image_MaxCorners_QualityLevel_MinDistance_BlockSize_UseHarris_t> Image_MaxCorners_QualityLevel_MinDistance_BlockSize_UseHarris;
+typedef std::tr1::tuple<string, int, double, int, int, bool> Image_MaxCorners_QualityLevel_MinDistance_BlockSize_gradientSize_UseHarris_t;
+typedef perf::TestBaseWithParam<Image_MaxCorners_QualityLevel_MinDistance_BlockSize_gradientSize_UseHarris_t> Image_MaxCorners_QualityLevel_MinDistance_BlockSize_gradientSize_UseHarris;
 
-PERF_TEST_P(Image_MaxCorners_QualityLevel_MinDistance_BlockSize_UseHarris, goodFeaturesToTrack,
+PERF_TEST_P(Image_MaxCorners_QualityLevel_MinDistance_BlockSize_gradientSize_UseHarris, goodFeaturesToTrack,
```

Comment:
You need to update sanity perf data as well: the name of test case was changed, that's why perf tests are failed.
There's need to change `testdata/perf/imgproc.xml` file. [Here](https://github.com/opencv/opencv/wiki/HowToUsePerfTests#how-to-update-perf-data) is the description of how to do that or you can change it manually.
Then you need to create a branch with updated file in your `opencv_extra` fork with the same name as current branch has (i.e. `goodFeaturesToTrack_added_gradiantSize`) and provide a PR to opencv_extra project.